<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1541229238933" stop="1541229238933">
  <description>This is the tests collection phase. Failures are modules that failed to collect.</description>
  <name>test_collection_phase</name>
  <title>Collection phase</title>
  <labels/>
  <test-cases>
    <test-case start="1541229238933" status="broken" stop="1541229238933">
      <name>test_login</name>
      <failure>
        <message>failed</message>
        <stack-trace>import file mismatch:
imported module 'test_login' has this __file__ attribute:
  F:\PythonItem\Appium\ZZ01\Day01\BaiNianAl\Scripts\test_login.py
which is not the same as the test file we want to collect:
  E:\软件测试\进阶课程\12，移动自动化项目\day02\04_代码\BaiNianAl\Scripts\test_login.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules</stack-trace>
      </failure>
      <attachments/>
      <labels/>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
